statsmodels/tsa/vector_ar/svar_model.py

```diff
@@ -586,6 +586,7 @@ def __init__(self, endog, endog_lagged, params, sigma_u, lag_order,
         else:
             trendorder = None
         self.k_trend = k_trend
+        self.k_exog = k_trend  # now required by VARProcess
```

Comment:
The "now" will be ambiguous in a few years.  Maybe "as of 0.9"?

Reply:
"now" in the repository can always be discovered in the blame view to check the commit that added or changed a line, as long as the history is not made noisy.